tests/test_endpoints.py

```diff
@@ -0,0 +1,236 @@
+from datetime import datetime
+
+import pytest
+
+TEST_PAGE_NAME = f"Test Page - {datetime.now()}"
+
+
+def compare_cleaned_ids(id1: str, id2: str) -> bool:
+    return id1.replace("-", "").strip() == id2.replace("-", "").strip()
+
+
+# PAGES
+# `test_pages_create` generates a temporary page and uses its ID
+# as input for all the other tests. Their objects (databases, blocks, comments)
+# will be created inside the page. With this method we have a
+# unique environment for each run, using the current datetime as identifier
+
+
+@pytest.fixture(scope="session")
+def test_pages_create(client, testing_page) -> str:
```

Comment:
I'd prefer that we keep the `test_` prefix for test functions, and use something that looks like a variable name for fixtures.